.circleci/config.yml

```diff
@@ -68,8 +72,77 @@ jobs:
           no_output_timeout: 20m
       - run: make ci-build-and-upload-cli
 
+  e2e-tests-aws:
+    docker:
+      - image: circleci/python:3.6
+    environment:
+      CORTEX_TEST_REALTIME_DEPLOY_TIMEOUT: 60
+      CORTEX_TEST_BATCH_DEPLOY_TIMEOUT: 30
+      CORTEX_TEST_BATCH_JOB_TIMEOUT: 200
+      NUM_BUILD_PROCS: 2
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - install-go
```

Comment:
Is go necessary?

Reply:
to build the CLI only, we can drop it if we use it the one in S3